internal/provider/features.go

```diff
@@ -255,6 +255,21 @@ func schemaFeatures(supportLegacyTestSuite bool) *pluginsdk.Schema {
 			},
 		},
 
+		"recovery_services_vault": {
```

Comment:
this should be plural 
```suggestion
		"recovery_services_vaults": {
```